index.js

```diff
@@ -1,27 +1,46 @@
 var chalk = require('chalk');
 
+var buildPolicyString = require('./lib/utils')['buildPolicyString'];
+
+var CSP_SELF        = "'self'";
+var CSP_NONE        = "'none'";
```

Comment:
Do not wrap them here though, as array are passed by reference and additional rules would add up.